i18n/en-US/core/commands.json

```diff
@@ -0,0 +1,13 @@
+{
+	"DESCRIPTION": "Receive a list of TypicalBot's commands.",
+	"USAGE": "commands",
+	"CHECK_DM": "A list of commands should have arrived in your Direct Messages.",
+	"TYPICAL_COMMANDS": "TypicalBot Commands",
+	"TEXT": "Use `{{prefix}}help <command>` to get more information on a specific command.",
+	"OWNER": "» Server Owner (Level 4)",
+	"ADMIN": "» Server Administrator (Level 3)",
+	"MOD": "» Server Moderator (Level 2)",
+	"DJ": "» Server DJ (Level 1)",
+	"MEMBER": "» Server Member (Level 0)",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.